guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,173 @@
+[id="Upgrading-Project-from-EL7-to-EL8_{context}"]
+= Upgrading {Project} from Enterprise Linux 7 to Enterprise Linux 8
+
+Use this procedure to upgrade your {Project} installation from Enterprise Linux 7 to Enterprise Linux 8.
+LEAPP is the supported tool for Enterprise Linux 7 to Enterprise Linux 8 upgrades.
+
+.Prerequisites
+* {Project} {ProjectVersionPrevious} running on CentOS Linux 7.
+* {Project} installations running on {RHEL} 7 cannot upgrade to {RHEL} 8.
+You must have a {RHEL} compatible LEAPP build to upgrade.
+* Access to available repositories or a local mirror of repositories.
+* No outstanding updates.
+* {Project} installations can be upgraded to CentOS Stream 8 and a {RHEL} rebuild.
+
+.Procedure
+[start=1]
```

Comment:
is there a way to avoid those `start=X` lines? it's horrible to read and will make us miss updating them if we ever change the procedure

Reply:
@evgeni I had to insert the `start=x` lines because the steps kept restarting at 1 throughout the document due to code blocks and other sentences interrupting the flow of the procedure.

Reply:
I super rarely use this. IMHO it's not needed here. Please remove all empty lines between list elements and remove all instances of "start=X".

Reply:
Can we compromise? I will only use `start=x` starting at the 8th step because I have tried playing around with the + sign and attempted dozens of build previews and it will not continue the numbering after the 7th step. The + signs end up in the text when I do a build preview and it doesn't matter where I put them. The numbering starts over at 1 after number 7.

Reply:
Another step was recommended and added. Now the `start=x` will begin at the 9th step if you guys are cool with it.

Reply:
Never mind. Vendula helped me fix the issue I was having. I will take out all `start=x` references.